python/list-streams/list_streams_lambda.py

```diff
@@ -0,0 +1,54 @@
+import boto3
+import time
+import random
+import botocore.exceptions
+
+retryable_exceptions = ['ClientLimitExceededException', 'InternalFailure']
+max_requests_per_second = 20
+
```

Comment:
recommendation is to use https://docs.aws.amazon.com/lambda/latest/dg/security-iam.html